src/schema/objects/datatypes.yaml

```diff
@@ -22,7 +22,7 @@ eeg:
   description: Electroencephalography
 ephys:
   value: ephys
-  display_name: Electrophysiology
+  display_name: Intracellular Electrophysiology
```

Comment:
this is wrong -- just reset --hard to prior commit and remove all mentioning of iephys:
- there is actually no ephys in BEP032 ATM
- ephys is about both types of recording -- above change makes no sense...

Reply:
done